File: guanabara-java/src/com/danillo/algoritmos/lista08/Exercicio83.java
package com.danillo.algoritmos.lista08;

import java.util.Random;

public class Exercicio83 {
    public static void main(String[] args){
        int[] vetor = new int[5];

        for(int i = 0; i < vetor.length; i++){
            Random aleatorio = new Random();
            int x = aleatorio.nextInt(99);
            vetor[i] = x;
        }

        System.out.println("Vetor desordenado: ");
        for(int i = 0; i < vetor.length; i++){
            System.out.println(vetor[i]);
        }

        int n = vetor.length;
        for (int i = 0; i < n - 1; i++) {
            for (int j = 0; j < n - i - 1; j++) {
                if (vetor[j] > vetor[j + 1]) {
                    int aux = vetor[j];
                    vetor[j] = vetor[j + 1];
                    vetor[j + 1] = aux;
                }
            }
        }

        System.out.println("Vetor ordenado: ");
        for(int i = 0; i < vetor.length; i++){
            System.out.println(vetor[i]);
        }
    }
}

File: guanabara-python/src/lista08/exercicio83.py
import random

vetor = []

for i in range(20):
    i = random.randint(0, 99)
    vetor.append(i)

print("Vetor desordenado: ", vetor)

for i in range(19):
    for j in range(19):
        if vetor[j] > vetor[j + 1]:
            aux = vetor[j]
            vetor[j] = vetor[j + 1]
            vetor[j + 1] = aux

print("Vetor ordenado: ", vetor)
